pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/nonpepperkit/NonPepperStatusKitService.java

```diff
@@ -0,0 +1,149 @@
+package org.broadinstitute.dsm.model.nonpepperkit;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.dao.bookmark.BookmarkDao;
+import org.broadinstitute.dsm.db.dao.kit.KitDao;
+import org.broadinstitute.dsm.db.dao.kit.KitDaoImpl;
+import org.broadinstitute.dsm.db.dao.user.UserDao;
+import org.broadinstitute.dsm.db.dto.bookmark.BookmarkDto;
+import org.broadinstitute.dsm.db.dto.user.UserDto;
+import org.broadinstitute.dsm.statics.DBConstants;
+
+@Slf4j
+public class NonPepperStatusKitService {
+    private static final String STATUS_KIT_THRESHOLD = "STATUS_KIT_THRESHOLD";
+    private KitDao kitDao;
+
+    public NonPepperStatusKitService() {
+        this.kitDao = new KitDaoImpl();
+    }
+
+    public KitResponse getKitsBasedOnStudyName(String studyGuid) {
+        if (StringUtils.isBlank(studyGuid)) {
+            return new KitResponseError(KitResponse.MISSING_STUDY_GUID);
```

Comment:
This would typically be a 400 error, but it looks like it probably returns a 200?
Likewise for the following checks.